### ISSUE TITLE
How can I use MySQL with utf-8?

### ISSUE THREAD

**Issue by izombielandgit** [privacyidea/privacyidea#1519]
I install privacyidea on Ubuntu 16.04 with "add-apt-repository ppa:privacyidea/privacyidea"

I found that the "pi" database use latin1, so I change it to utf-8 (database and tables). 
Then I tried to use the "edit user", it warned that "Failed to update user."

Then I create a new database with utf-8, change the "/etc/privacyidea/pi.cfg", and run "pi-manage createdb". It got the same problem.

By the way, How can I install privacyidea on Ubuntu 18.04 like "add-apt-repository ppa:privacyidea/privacyidea"

Thank you

**Comment by izombielandgit** [comment]
I install privacyidea on Ubuntu 16.04 with "add-apt-repository ppa:privacyidea/privacyidea"

I found that the "pi" database use latin1, so I change it to utf-8 (database and tables). 
Then I tried to use the "edit user", it warned that "Failed to update user."

Then I create a new database with utf-8, change the "/etc/privacyidea/pi.cfg", and run "pi-manage createdb". It got the same problem.

By the way, How can I install privacyidea on Ubuntu 18.04 with "add-apt-repository ppa:privacyidea/privacyidea"

Thank you
